build_tools/python/benchmarks/iree/definitions.py

```diff
@@ -16,6 +16,24 @@
 MODULE_BENCHMARK_TOOL = "iree-benchmark-module"
 
 
+def _generate_run_specs(
+    compile_specs: Sequence[iree_definitions.CompileSpec],
+    run_configs: Sequence[iree_definitions.RunConfig],
+    device_spec: common_definitions.DeviceSpec,
+    input_data: common_definitions.ModelInputData = common_definitions.
+    RANDOM_MODEL_INPUT_DATA,
+) -> List[iree_definitions.RunSpec]:
+  """Generates the run specs from the product of compile specs and run configs.
```

Comment:
Aside, not this PR: the "run spec" and "run config" distinction seems like weird naming to me. If you asked me which was which, I don't think I could say. Ideas:

1. "model run spec" for "compile spec" + "run spec", since the info needed to run an entire model requires compilation and running?
2. e2e spec
3. One of the above plus call the bit that is only about running a compiled model an "IREE run spec" or "VMFB run spec"

Reply:
Will do in another change.